mne/io/egi/tests/test_egi.py

```diff
@@ -58,6 +58,9 @@
 egi_pause_w1337_events = None
 egi_pause_w1337_skips = [(21956000.0, 40444000.0), (60936000.0, 89332000.0)]
 
+# TODO: Remove once complete deprecation / FutureWarning about events_as_annonations
+pytestmark = pytest.mark.filterwarnings("ignore:.*events_as_annotation.*:FutureWarning")
```

Comment:
I think maybe this got defined but wasn't used to decorate anything (hence the CI errors)?

Reply:
Actually it is automagically used in this module

https://docs.pytest.org/en/latest/how-to/skipping.html#skip-all-test-functions-of-a-class-or-module

Just need to add it in `test_montage.py` (where I'll just decorate the one failing func)